enhancements/fbc-run-bundle.md

```diff
@@ -0,0 +1,268 @@
+---
+title: fbc-run-bundle
+authors:
+  - "@rashmigottipati"
+reviewers:
+  - "@joelanford"
+  - "@jmrodri"
+  - "@VenkatRamaraju"
+approvers:
+  - "@joelanford"
+  - "@jmrodri"
+  - "@VenkatRamaraju"
+creation-date: 2022-04-20
+last-updated: 2022-04-20
+status: implementable
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Add FBC support to run bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+- Do we need to build a File-Based Catalog to run this command?
+
+  - We can use `quay.io/operatorhubio/catalog:latest` for testing and generate a File-Based Catalog at runtime when the run bundle command is invoked.
+
+- Do we need to make our own image to use as a default one for run bundle and run bundle-upgrade?
+
+  - Currently, we are using `quay.io/operator-framework/opm:latest` as the default index image. We might have to build an image similar to `quay.io/operatorframework/catalog:latest` which uses opm as the base image and also has a built catalog in it.
+
```

Comment:
Right, that's for the FBC case. We are still using `quay.io/operator-framework/opm:latest` as the base image for the SQLite images. I don't think we need a default empty catalog for the FBC images as we are building our own.

Reply:
> I don't think we are using the quay.io/operator-framework/opm:latest anywhere (correct me if I'm wrong)

Don't we use this as the `image` in the catalog pod we run for FBC?

Reply:
@joelanford yeah, I meant apart from containing the opm binary, this image has no other value, correct? Since we're generating our own FBC, I don't think we need a new default image that contains both the opm binary and an empty catalog. An image with _only_ the former will suffice.

Reply:
@joelanford Yea, this is the image in the catalog pod we run for FBC as well. 

@VenkatRamaraju Yup! We don't need a new default image that has both opm and a minimum catalog since we are always going to be build an FBC if an index isn't provided.